platform/server-bundle/src/main/resources/task-channel.json

```diff
@@ -0,0 +1,24 @@
+{
+  "displayName": "logger",
+  "triggerTaskEvents" : [],
+  "actionTaskEvents": [
+    {
+      "serviceInterface": "org.motechproject.server.service.LoggerService",
+      "serviceMethod": "logMessage",
+      "displayName": "logger.log",
+      "subject": "LogLogs",
```

Comment:
this is redundant, there are two main ways for creating task actions. First using serviceInterface (the one you used here), the second using events (for example in email module). Field subject is for the second option, so here you can remove this